Do not install midi version of the intro

zezinho42 opened the pull request:
The midi file is not used by Emilia, and some dependencies checkers wrongly pull timidity because it is there. At least for Mageia ;-)

sergiomb2 commented:
Added Gentoo gcc-4.6 patch is already in code , but in begin of includes , I had to revert merge

sergiomb2 committed:
iii

sergiomb2 committed:
Add License.txt

zezinho42 committed:
Do not install midi version of the intro

The midi file is not used by Emilia, and some dependencies checkers wrongly pull timidity because it is there. At least for Mageia ;-)

sergiomb2 commented:
in Fedora we got Add Requires: timidity++-patches so that the music works
house use it : 
data/house/pinball.pbl
1723:      music intro.mid

zezinho42 commented:
Ok for house table, but this commit is about the midi file in the tux table!

sergiomb2 commented:
cherry-pick 96c8eaf  to devel branch 

hash updated again [1]

[1]
https://github.com/sergiomb2/pinball/commit/a2833450e9e51d4322935271439e1050705c4b6b

sergiomb2 commented:
yet another hash for this cherry-pick 
https://github.com/sergiomb2/pinball/commit/32f45ad1a8d84e73759840157e8e66c4448a0e33
